src/ts/tests/snake.test.ts

```diff
@@ -8,7 +8,7 @@ const {
   move_snake, 
   reflect_point,
   grow_snake,
-  eq_location } = require('../src/snake.ts');
+  eq_location } = require('../snake.ts');
```

Comment:
typescript wants you to require `snake` rather than `snake.ts`

![image](https://user-images.githubusercontent.com/77482/102694616-191c6f00-41d7-11eb-9ca2-9438c154ad82.png)